pyteal/ast/itxn.py

```diff
@@ -253,10 +253,10 @@ def SetFields(cls, fields: dict[TxnField, Expr | list[Expr]]) -> Expr:
     def ExecuteMethodCall(
         cls,
         *,
-        app_id: Expr,
+        app_id: Optional[Expr],
```

Comment:
```suggestion
        app_id: Expr | None,
```

Do you want to allow default `= None` or have the user explicitly provide `None` ?